docs/deployment.md

```diff
@@ -55,9 +63,11 @@ If you are using a static inventory run:
 
 If you provisioned your servers using the OpenStack playbook and are using the IDR dynamic inventory run:
 
-    ansible-playbook -i ../inventories/openstack-idr.py --diff -u centos idr-00-preinstall.yml idr-01-install-idr.yml idr-03-postinstall.yml
+    ansible-playbook -i ../inventories/openstack-idr.py --diff -u centos idr-00-preinstall.yml idr-01-install-idr.yml idr-02-services.yml idr-03-postinstall.yml
```

Comment:
I'd very much like to get away from these list of playbooks. I could imagine a top-level (i.e. above `ansible/`) playbook like `deploy-core.yml`

Reply:
Let's discuss this in a couple of weeks since it ties in with the naming and deployment of all the separate components.